packages/core/src/components/components.md

```diff
@@ -11,27 +13,49 @@
 @page collapse
 @page collapsible-list
 @page context-menu
-@page dialog
 @page editable-text
-@page forms
-@page html
 @page hotkeys
 @page icon
 @page menu
 @page navbar
 @page non-ideal-state
-@page overlay
-@page popover
-@page portal
 @page progress-bar
 @page skeleton
 @page spinner
-@page sliders
 @page table-html
```

Comment:
kinda feel like we should include this in `HTML elements` then

Reply:
oh i disagree, it does much more than those components. has its own modifiers!

Reply:
ok cool